repaygithub/cactus
# CACTUS-428 :: IconButton Focus

Ticket: https://repayonline.atlassian.net/browse/CACTUS-428

It kind of sucks that I ended up having to make changes to `DateInput` in order to get the integration tests to pass, but as far as I can tell the changes aren't causing problems. I'm not completely sure why adding an `:after` pseudo element caused tests to fail in FireFox in the first place, but it did, and that's how I was able to get around it. 🤷‍♂️ 

### Testing
1. Bring up the `IconButton` story and focus on the button to make sure the outline appears.
2. Change the `IconButton` size and the shape in the theme and make sure it looks good with different shapes & sizes.
3. Repeat steps 1 & 2 in IE11 to double check that it works there too, because I did have to do a workaround for IE.
4. Go to the `DateInput` story and make sure that works too -- specifically when you click the calendar icon to open the popup.
